packages/web-client/app/components/card-pay/layer-connect-modal/index.hbs

```diff
@@ -55,7 +55,6 @@
         <CardPay::LayerTwoConnectCard
           class="layer-connect-modal__wrapped-card"
           @suppressHeader={{true}}
-          @includeExplanations={{true}}
```

Comment:
This conditional was hard-coded and the explanation isn’t shown in the design, so I’ve removed it.